client/landing/stepper/hooks/use-is-site-big-sky-eligible.ts

```diff
@@ -1,30 +1,62 @@
 import config from '@automattic/calypso-config';
-import { isPremiumPlan } from '@automattic/calypso-products';
+import { isBusinessPlan, isPremiumPlan } from '@automattic/calypso-products';
 import { useBreakpoint } from '@automattic/viewport-react';
 import { useSelect } from '@wordpress/data';
+import { useState, useEffect } from 'react';
+import { useExperiment } from 'calypso/lib/explat';
 import { useIsSiteOwner } from '../hooks/use-is-site-owner';
 import { ONBOARD_STORE } from '../stores';
 import { useSite } from './use-site';
 import type { OnboardSelect } from '@automattic/data-stores';
 
-export function useIsBigSkyEligible(): boolean | null {
+const bypassBigSkyExperiment = () => config.isEnabled( 'big-sky' );
```

Comment:
No need for the callback. 
```suggestion
const bypassBigSkyExperiment = config.isEnabled( 'big-sky' );
```